tests/pytests/test_aggregate.py

```diff
@@ -149,16 +150,32 @@ def testStdDev(self):
         self.env.assertEqual(29, int(float(row['avgPrice'])))
 
     def testParseTime(self):
+        distro_name = distro.name().lower()
+
+        expected = ['brand', '', 'count', '1518', 'dt', '2018-01-31T16:45:44Z',
+                    'parsed_dt', '1517417144']
+
+        # Skip on Alpine Linux, as its strptime() doesn't support '%FT%TZ' format
+        if distro_name != 'alpine linux':
+            cmd = ['FT.AGGREGATE', 'games', '*',
+                'GROUPBY', '1', '@brand',
+                'REDUCE', 'COUNT', '0', 'AS', 'count',
+                'APPLY', 'timefmt(1517417144)', 'AS', 'dt',
+                'APPLY', 'parsetime(@dt, "%FT%TZ")', 'as', 'parsed_dt',
+                'LIMIT', '0', '1']
+            res = self.env.cmd(*cmd)
+
+            self.env.assertEqual(expected, res[1])
+        
         cmd = ['FT.AGGREGATE', 'games', '*',
-               'GROUPBY', '1', '@brand',
-               'REDUCE', 'COUNT', '0', 'AS', 'count',
-               'APPLY', 'timefmt(1517417144)', 'AS', 'dt',
-               'APPLY', 'parsetime(@dt, "%FT%TZ")', 'as', 'parsed_dt',
-               'LIMIT', '0', '1']
+                'GROUPBY', '1', '@brand',
+                'REDUCE', 'COUNT', '0', 'AS', 'count',
+                'APPLY', 'timefmt(1517417144)', 'AS', 'dt',
+                'APPLY', 'parsetime(@dt, "%Y-%m-%dT%H:%M:%SZ")', 'as',
```

Comment:
Add a comment explaining this is testing an equivalent longer format in addition to the shorter format '%FT%TZ' not supported on Alpine Linux